### ISSUE TITLE
latest react gives error on unknown props

### ISSUE THREAD

**Issue by dlebedynskyi** [react-toolbox/react-toolbox#607]
react 15.2.0 added 
`Add warning for unknown properties on DOM elements` 

this introduces a lot of warnings in console like 
```
warning.js:44 Warning: Unknown prop `composeTheme` on <button> tag. Remove this prop from the element. For details, see https://fb.me/react-unknown-prop
    in button (created by IconButton)
```
```
Warning: Unknown prop `minWidth` on <input> tag. Remove this prop from the element. For details, see https://fb.me/react-unknown-prop
    in input (created by AutosizeInput)
```
add so on.

**Comment by emirotin** [comment]
Relevant: https://github.com/Hacker0x01/react-datepicker/issues/517#issuecomment-230119718

**Comment by oliviertassinari** [comment]
If that help. That's how we have solved some of the warnings https://github.com/callemall/material-ui/pull/4607/files.

**Comment by javivelasco** [comment]
Thank you @oliviertassinari !! I've just [removed the prop spreading](https://github.com/javivelasco/react-css-themr/commit/f0225cdbfbe6b6a559382e8aa1c98324316bf9f2) in react-css-themr and also I'm going to update to see how many components are broken.

**Comment by BaoDelta** [comment]
Hi @javivelasco, can you publish a new version to npm?

**Comment by javivelasco** [comment]
Today, I wanted to fix a couple of things more

**Comment by BaoDelta** [comment]
@javivelasco I think I was too eager. Anyway, thanks for your hard work.

**Comment by albertotorresfoltyn** [comment]
inputs from material-formsy seems to have a related prooblem :D

**Comment by gabrielenosso** [comment]
Still no publish on npm?

**Comment by javivelasco** [comment]
It's published as 1.0.2. I've removed all unknown props I saw. The testing view is not giving any problems and neither the documentation. From this point it somebody is experiencing this problem probably it's because she's passing unknown props down to React Toolbox components!
